test/models/comment_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+
+class CommentTest < ActiveSupport::TestCase
+  def setup
+    @user = users(:musa)
+    @post = posts(:sample_post1)
+    @comment = Comment.new(content: 'Lorem ipsum', user_id: @user.id,\
+                           post_id: @post.id)
+  end
+
+  test 'should be valid' do
+    assert @comment.valid?
+  end
+
+  test 'user id should be present' do
```

Comment:
We can safely remove testing for the presence of user_id as the association test below takes care of this already